RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress.swift

```diff
@@ -256,6 +259,20 @@ struct AccountRecoveryScanInProgress: Sendable, FeatureReducer {
 	func reduceDismissedDestination(into state: inout State) -> Effect<Action> {
 		.run { _ in await dismiss() }
 	}
+
+	private func getAccountAppearanceID(
+		mode: State.Mode,
+		offset: Int,
+		networkID: NetworkID
+	) async -> AppearanceID {
+		switch mode {
+		case .createProfile:
+			return AppearanceID.fromNumberOfAccounts(offset)
```

Comment:
@CyonAlexRDX is this assumption correct? Seems to be, but not 100% sure.

Reply:
I think that looks good